_config/site.yml

```diff
@@ -56,6 +56,8 @@ profiles:
     dcp_base_protocol_relative_url: //dcpbeta-searchisko.rhcloud.com/
     dcp2_base_url: http://dcp2-searchisko.rhcloud.com/
     dcp2_base_protocol_relative_url: //dcp2-searchisko.rhcloud.com/
+    keycloak_account_url: https://it-developers.stage.redhat.com/auth/realms/rhd/account/
+    keycloak_auth_url: https://it-developers.stage.redhat.com/auth/
```

Comment:
And that is blocked on the merging: https://github.com/redhat-developer/developers.redhat.com/pull/332, which will hopefully happen this week.